test/safe/branch.test.ts

```diff
@@ -256,8 +236,9 @@ describe("Branch Webhook", () => {
 			});
 		});
 
-		it("should not update Jira issue if there are no issue Keys in the branch name", async () => {
+		it.skip("should not update Jira issue if there are no issue Keys in the branch name", async () => {
```

Comment:
Why are we skipping rather than deleting these? Or fixing them?

Reply:
It is a strong no go. We shuldn't skip working tests.

We went through this once before and caused an incident

Reply:
I didn't write them originally and this PR is big enough (and my enough of my time used) without having to fix other people's tests.  We could delete them, but I think the idea was that this was a test to be implemented by someone but never finished.

Reply:
I wonder why did we have to skip it? Was it failing?

The test-case makes sense to me, we are trying to send a branch without issue keys and make sure we don't call Jira.


However, I think you are right that the test wouldn't work. Now when we have a branch queue it can give false-negatives, because we don't wait until the queue message will be processed. We should make it as a unit test for the queue listener. Which is out of scope of this PR.

Reply:
Let's leave a not about it (to move them to unit tests and reason why)

Reply:
I'm skipping it because it literally does nothing.  It creates a mock function that can't ever be called, then asserting that it's never called.  We're not asserting anything of value, and as such, this test will pass no matter what happens.  I'm skipping it because it's a reminder that this test needs to be fixed.

Reply:
It could happen when we were migrating to branches queue. Before this test would work, because it would fail if we try making request to Jira (and we also try to fetch the latest commit in the handler, if I am not wrong)